src/sentry/conf/server.py

```diff
@@ -1068,6 +1069,7 @@ def create_partitioned_queues(name):
         ('org:read', 'Read access to organization details.'),
     ), (
         ('org:integrations', 'Read, write, and admin access to organization integrations.'),
+        ('project:integrations', 'Read, write, and admin access to project integrations.'),
```

Comment:
Is this where this goes?

Reply:
seems right. you should probably also add here https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/constants/index.jsx